acceleration/README.md

```diff
@@ -0,0 +1,168 @@
+# Sample
+
+This sample showcases the way to use spiced to simplify the process of building a CQRS pattern for your application.
+
+In this sample, we have a `/orders` API which creates a random order with a random count.
```

Comment:
Does this sample need to call the `/orders` API for it to work? If so, we should show that step here (or even better, just pre-populated on load)

Reply:
Please resolve / address in next iteration.  Closing comment to merge.